flink-connector-postgres-cdc/src/main/java/io/debezium/connector/postgresql/connection/PostgresConnection.java

```diff
@@ -53,6 +52,8 @@
 /**
  * {@link JdbcConnection} connection extension used for connecting to Postgres instances.
  *
+ * <p>Line 755 : cherry pick from DBZ-5470
```

Comment:
Add: Remove it when debezium version is upgraded above 2.0.0.Final.

Reply:
done